examples/shell/mbed/ChipShellCollection.h

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+extern "C" {
+// A list of shell commands provided by ChipShell
+void cmd_base64_init(void);
+void cmd_btp_init(void);
+void cmd_device_init(void);
+void cmd_misc_init(void);
+void cmd_otcli_init(void);
+void cmd_mbed_utils_init(void);
```

Comment:
I'm not sure if we have to create ChipShellCollection.h file one more time. Maybe just create mbed-specific commands file i.e ChipShellMbedCollection.h with cmd_mbed_utils_init() and in main.cpp include both.

Reply:
Good idea! I updated my PR with this suggestion.